models/model.js

```diff
@@ -0,0 +1,34 @@
+const db = require('../db/connection');
+// dont think this is what i need?
+
+const fetchTopics = () => {
+    return db.query(
+        `
+        SELECT * FROM topics;
+        
+        `
+    )
+    .then(result => {
+        console.log('model',result.rows)
+        const topics = result.rows
+        if (result.rows.length === 0) {
+            return Promise.reject({status: 404, msg: 'resource not found'});
```

Comment:
We can remove this line and instead include error handling for an invalid path (404) in our app.js - so long as we have still have the catch in our controller file.